src/model/text/Comment.java

```diff
@@ -4,8 +4,18 @@
 
     private String articleID;
 
-    public Comment(String content, String author, String articleID) {
+    public String getArticleID() {
+		return articleID;
+	}
+
+	public void setArticleID(String articleID) {
+		this.articleID = articleID;
+	}
+
+	public Comment(String content, String author, String articleID) {
```

Comment:
Please keep constructor as first method in any Java class, it's actually coding style in Java.